src/utils/channel_utils.js

```diff
@@ -656,3 +656,10 @@ export function filterChannelsMatchingTerm(channels: Array<Channel>, term: strin
             displayName.startsWith(lowercasedTerm);
     });
 }
+
+export function groupChannelMatchesTerm(groupChannel: Channel, term: string): boolean {
```

Comment:
Why not follow the previous naming convention? something like `filterGroupChannelMatchingTerm`

Reply:
Makes sense not only with the name, but the implementation too :)